src/d3-funnel/d3-funnel.js

```diff
@@ -738,12 +844,39 @@ class D3Funnel {
 	}
 
 	/**
+	 * @param {int} index
+	 *
+	 * @return {string}
+	 */
+	_getOverlayPathDefinition(index) {
+		const commands = [];
+
+		this.overlayPaths[index].forEach((command) => {
+			commands.push([command[2], command[0], command[1]]);
+		});
+
+		return this.navigator.plot(commands);
+	}
+
+	/**
 	 * @param {Object} data
 	 *
 	 * @return {void}
 	 */
 	_onMouseOver(data) {
-		d3.select(d3.event.target).attr('fill', this.colorizer.shade(data.fill.raw, -0.2));
+		const children = d3.event.target.parentElement.childNodes;
```

Comment:
hover effects are applied to all paths within a block. It could be either a single path or background + value overlay paths